src/components/Clock.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type Props = {
+  clockName: number;
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends React.Component<Props> {
+  state: State = {
+    date: new Date(),
+  };
+
+  watch = setInterval(() => {
+    this.setState({ date: new Date() });
+  }, 1000);
```

Comment:
it is needed to setInterval in a componentDidMount method. And use componentDidUpdate method for this
![image](https://user-images.githubusercontent.com/64437769/133227884-27986025-b075-4b07-a100-1121a6649098.png)